src/common/MessageKeys.ts

```diff
@@ -1,4 +1,5 @@
 export enum MessageKeys {
 	EDIT = 'wikibase-edit',
 	ALIAS_SEPARATOR = 'wikibase-aliases-separator',
+	ALL_LANG = 'wikibase-entitytermsforlanguagelistview-more',
```

Comment:
Please let's not use abbreviations, it's a slippery slope.

Reply:
changed